docs/installation.rst

```diff
@@ -45,54 +45,106 @@ Fedora 22
 .. _installing node.js on Red Hat: https://nodejs.org/en/download/package-manager/#enterprise-linux-and-fedora
 
 
+Conda
+^^^^^
+
+Conda installation and conda environment setup should be the first step in
+creating a development environment for Minerva.
+
+- Install Conda (https://conda.io/miniconda.html)
+
+- Create minerva development environment
+
+::
+
+    mkdir geoviz
+    git clone https://github.com/Kitware/minerva.git
+    cd minerva
+    conda env create -f conda_env.yml python=2.7
+    source activate minerva-dev
+
+- Run mongo daemon
+
+::
+
+    mongod &
```

Comment:
With a clean Ubuntu, I need to create a directory for mongo frist then run
**mkdir mongo_data
mongod --dbpath mongo_data &**

Reply:
Fixed